app/models/item.rb

```diff
@@ -3,4 +3,10 @@ class Item < ApplicationRecord
   has_many :rentals
   has_many :pictures
   has_many :reviews, through: :rentals
+
+  validates :price_per_day, presence: true
+  validates :product_description, length: {minimum: 15, maximum: 500}, presence: true
+  validates :equipment_category, presence: true
+  validates :user_description, length: {minimum: 15, maximum: 500}, presence: true
+  validates :condition, presence: true
```

Comment:
validates :condition, inclusion: { in: ('Poor', 'Sufficient', 'Good', 'As new') }